osf/utils/migrations.py

```diff
@@ -176,6 +191,9 @@ def ensure_schemas(*args):
         if created:
             logger.info('Added schema {} to the database'.format(schema['name']))
 
+        schema_obj.registration_responses_jsonschema = build_flattened_jsonschema(schema_obj)
+        schema_obj.save()
```

Comment:
if RegistrationSchemas are added or updated, the cached jsonschema dict will be added/updated as well. (when `ensure_schemas` is run